### PR TITLE
Translate approval2template to Portuguese

### PR DESCRIPTION
**Pull request by maelle**
@pedropark99

### CODE REVIEW

**Comment by pedropark99** [comment]
@maelle my knowledges of Git might be failling with me here, but let's see if I did it right. The first thing I did, was to update my forked version of dev_guide, using git fetch and git rebase, with these commands:

```
git remote add upstream https://github.com/ropensci/dev_guide.git
git fetch upstream
git rebase upstream/main
```

Excellent, now the local copy of my forked repository is now in sync with the main repo (i.e. `ropensci/dev_guide`). Then I switched to branch `approval2template-pt` to start the review. Made some changes, commiteed. However, I am not authorized to write in any branch of the main repo, so when I tried to push these changes to this branch `approval2template-pt`, I got an error.


image

Am I doing this right? Should I make the changes in a separate branch on my fork, then, I create a PR to merge the changes of this separate branch in my fork into the branch `approval2template-pt` on the main repo, before we actually merge this specific PR?

**Comment by maelle** [comment]
> Should I make the changes in a separate branch on my fork, then, I create a PR to merge the changes of this separate branch in my fork into the branch approval2template-pt on the main repo, before we actually merge this specific PR?

Yes! Thanks for your patience and for helping us clarify these things.

@yabellini where is this best documented? https://translationguide.ropensci.org/es/howtoreview.es.html#tech?

**Comment by pedropark99** [comment]
Changes for review 1 at #679

**Comment by pedropark99** [comment]
@marianambc  @danielvartan   @joaojcorreia  this PR still needs an review 2.

Remember, if you have any doubts about the process, do not be afraid to ask to help here. @marianambc I am not sure if you saw one my comments where I explain how this review process works. If you did not read yet the Translation Guidelines of ROpenSci, I strongly recommend you to read it before engaging in the review. Link to guidelines:

https://translationguide.ropensci.org/

**Comment by danielvartan** [comment]
On it @pedropark99 .

**Comment by danielvartan** [comment]
I have completed the second review and opened a PR. @pedropark99, please check if everything is in order when you have some time.

**Comment by pedropark99** [comment]
Thank you very much for your help @danielvartan !

**Comment by pedropark99** [comment]
@yabellini  I'm confused, a GitHub notification popped up here about someone mentioning me in this PR, but I found no mentioning. Looks like to me that someone is asking for me to review this PR. Is that right?

**Comment by yabellini** [comment]
Hi @pedropark99, I'm merging all the finished reviews, so you probably will get many notifications today. I will also create a new PR for news reviews and ping you for the reviews. I understand this one already has the two reviews; is that right?

File: approval2template.pt.Rmd [hunk 1]
```diff
@@ -1,16 +1,16 @@
 ---
-repo-actions: true
+repo-actions: yes
```
Comment (yabellini) [inline approval2template.pt.Rmd:2]:
```suggestion
repo-actions: true
```

**Pull request merged**